src/roles/vrs-deploy/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 - include_role:
     name: skip-actions
   vars:
```

Comment:
Skip should be for only predeploy.. The correct fix is remove skip actions step here